pom.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks like you have added a byte order mark. Any reason for that?

```
$ hexdump -n 3 -C pom.xml 
00000000  ef bb bf                                          |...|
```

before the change:

```
$ hexdump -n 3 -C pom.xml 
00000000  3c 3f 78                                          |<?x|
```